### PR TITLE
Add routes for session CRUD

### PR DESCRIPTION
**Pull request by quickskape**
- Enable CORS to support debugging across devices
- Add shortid to generate session id
- Add basic session store

Link with https://github.com/tsangste/ChaosSpankr.Home/issues/5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-    "keywords": [
-        "ChaoSpankr"
-    ],
-    "author": "JJLPRS",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/tsangste/ChaosSpankr.Home/issues"
-    },
-    "homepage": "https://github.com/tsangste/ChaosSpankr.Server#readme"
+  "name": "chaosspankr",
```
Comment (tsangste) [inline package.json]: lol tabs again?

Comment (quickskape) [inline package.json]: Fixed, sorry.

File: routes/sessions.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    let state = req.body.state || ''
+
+    if (sessionId === '') {
+        res.status(500)
+        res.json({ message: "No session id received" })
+
+        return
+    }
+
+    var isValidState = sessionConstants.VALID_STATES.indexOf(state) > -1
```
Comment (tsangste) [inline routes/sessions.js]: i see a var!

File: routes/sessions.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+})
+
+/*
+    Delete a session
+    exampleCall: curl -H "Content-Type: application/json" -X DELETE -d '{"sessionId": "H1tSBr9F"}' http://localhost:3000/sessions/
+ */
+router.delete('/', (req, res) => {
+    let sessionId = req.body.sessionId
+
+    console.log()
```
Comment (tsangste) [inline routes/sessions.js]: empty log?

**Pull request merged**